tests/storage-spec.js

```diff
@@ -1,5 +1,41 @@
 import '../app/scripts/src/main';
 
+describe('windowReload', function () {
```

Comment:
Perhaps, but it's not in a separate file in the src. Don't think it matters much since it's quite small